### PR TITLE
adding a paragraph about A13

### PR DESCRIPTION
**Pull request by MAETempels**

### CODE REVIEW

**Comment by MAETempels** [comment]
In the TOC the link to H18 did not work. I hope, I have corrected this.
All added and changed links worked correctly in github.

File: docs/index.md [hunk 1]
```diff
@@ -413,13 +413,27 @@ a different working order is required.
...
 
 
-### <a name="H18-variants"/>H18 &mdash; two patterns
+### <a name="h18variations"/>H18 &mdash; two patterns
 
 [![](w/page241a.gif)][P241]      
 
 The work consists of two parts, starting with [Ground a][H18a], and changing halfway to [Ground b][H18b].   
 
+## Three&dash;paired joins
```
Comment (jo-pol) [inline docs/index.md]:
```suggestion
## Three paired joins   
```

File: docs/index.md [hunk 2]
```diff
@@ -413,13 +413,27 @@ a different working order is required.
...
+### <a name="h18variations"/>H18 &mdash; two patterns
 
 [![](w/page241a.gif)][P241]      
 
 The work consists of two parts, starting with [Ground a][H18a], and changing halfway to [Ground b][H18b].   
 
+## Three&dash;paired joins   
 
+Groundforge can not do real three&dash;paired joins. However, we can simulate such a join by treating it as a series of stitches. See [MAE-GF][p-mae-gf-3] for some examples.      
+### <a name="a13joins"/>A13 &mdash; examples of three&dash;paired joins
```
Comment (jo-pol) [inline docs/index.md]: I would expect this one in the side bar too. Subsequent sections don't follow the proper hyrarchy in the sidebar.

I created a sneak preview for you on https://jo-pol.github.io/gw-lace-to-gf/
You can paste the content of the sidebar in a local html file for some testing.

**Review by jo-pol — COMMENTED** [review]
Good initiative to add the patterns with 3-pair stitches. Not sure there are more of those.

File: docs/index.md [hunk 3]
```diff
@@ -130,12 +130,12 @@ title: Whiting index
...
 |  12 | [diagr][A12] [![](w/page84a.gif)][P84] | [N4](#n4) [![](w/page105a.gif)][P105] | [diagr][C12] [![](w/page126a.gif)][P126] | [diagr][D12] [![](w/page145a.gif)][P145] | [diagr][E12] [![](w/page167a.gif)][P167] | [diagr][F12] [![](w/page190a.gif)][P190] | [diagr][G12] [![](w/page210a.gif)][P210] | [diagr][H12] [![](w/page232a.gif)][P232] |
 |     | **A** | **B** | **C** | **D** | **E** | **F** | **G** | **H** |
-|  13 | [N3](#n3) [![](w/page85a.gif)][P85] | [N3](#n3) [![](w/page106a.gif)][P106] | [N3](#n3) [![](w/page128a.gif)][P128] | [N3](#n3) [![](w/page147a.gif)][P147] | [N3](#n3) [![](w/page169a.gif)][P169] | [N3](#n3) [![](w/page192a.gif)][P192] | [N3](#n3) [![](w/page211a.gif)][P211] | [N3](#n3)<br> [![](w/page234a.gif)][P234] |
+|  13 | [N3](#n3) [3 pairs](#a13joins) [![](w/page85a.gif)][P85] | [N3](#n3) [![](w/page106a.gif)][P106] | [N3](#n3) [![](w/page128a.gif)][P128] | [N3](#n3) [![](w/page147a.gif)][P147] | [N3](#n3) [![](w/page169a.gif)][P169] | [N3](#n3) [![](w/page192a.gif)][P192] | [N3](#n3) [![](w/page211a.gif)][P211] | [N3](#n3)<br> [![](w/page234a.gif)][P234] |
```
Comment (jo-pol) [inline docs/index.md]:
```suggestion
|  13 | [3 pairs](#a13joins) [![](w/page85a.gif)][P85] | [N3](#n3) [![](w/page106a.gif)][P106] | [N3](#n3) [![](w/page128a.gif)][P128] | [N3](#n3) [![](w/page147a.gif)][P147] | [N3](#n3) [![](w/page169a.gif)][P169] | [N3](#n3) [![](w/page192a.gif)][P192] | [N3](#n3) [![](w/page211a.gif)][P211] | [N3](#n3)<br> [![](w/page234a.gif)][P234] |
```

File: docs/index.md (hunk 2 above)
Comment (MAETempels) [reply docs/index.md]: I changed the side-bar (and corrected H18). Its in the first set of commits.
Will have to change it again as the title of the paragraph changes.

Comment (MAETempels) [reply docs/index.md]: Forgot a ' " '....

File: docs/index.md (hunk 1 above)
Comment (MAETempels) [reply docs/index.md]: Done

**Comment by MAETempels** [comment]
We could do some research if there are more three paired joins in the Whiting index. Maybe Gibritte has already done this. 
A quick glance: A4, B4, don't see more. 
When all are discovered, we can replace "N3" with a dedicated number for them.

Notice a ", " after G10 ....

File: docs/index.md (hunk 2 above)
Comment (MAETempels) [reply docs/index.md]: Notice: I can test the side bar by copying the table in my test.md. 
The preview shows the layout.

File: docs/index.md [hunk 4]
```diff
@@ -126,16 +126,16 @@ title: Whiting index
...
 |  12 | [diagr][A12] [![](w/page84a.gif)][P84] | [N4](#n4) [![](w/page105a.gif)][P105] | [diagr][C12] [![](w/page126a.gif)][P126] | [diagr][D12] [![](w/page145a.gif)][P145] | [diagr][E12] [![](w/page167a.gif)][P167] | [diagr][F12] [![](w/page190a.gif)][P190] | [diagr][G12] [![](w/page210a.gif)][P210] | [diagr][H12] [![](w/page232a.gif)][P232] |
 |     | **A** | **B** | **C** | **D** | **E** | **F** | **G** | **H** |
-|  13 | [N3](#n3) [![](w/page85a.gif)][P85] | [N3](#n3) [![](w/page106a.gif)][P106] | [N3](#n3) [![](w/page128a.gif)][P128] | [N3](#n3) [![](w/page147a.gif)][P147] | [N3](#n3) [![](w/page169a.gif)][P169] | [N3](#n3) [![](w/page192a.gif)][P192] | [N3](#n3) [![](w/page211a.gif)][P211] | [N3](#n3)<br> [![](w/page234a.gif)][P234] |
+|  13 | [N3](#n3), [3 pairs](#a13joins) [![](w/page85a.gif)][P85] | [N3](#n3) [![](w/page106a.gif)][P106] | [N3](#n3) [![](w/page128a.gif)][P128] | [N3](#n3) [![](w/page147a.gif)][P147] | [N3](#n3) [![](w/page169a.gif)][P169] | [N3](#n3) [![](w/page192a.gif)][P192] | [N3](#n3) [![](w/page211a.gif)][P211] | [N3](#n3)<br> [![](w/page234a.gif)][P234] |
```
Comment (jo-pol) [inline docs/index.md]: I don't think the link to N3 is needed any more.

```suggestion
|  13 | [3 pairs](#a13joins) [![](w/page85a.gif)][P85] | [N3](#n3) [![](w/page106a.gif)][P106] | [N3](#n3) [![](w/page128a.gif)][P128] | [N3](#n3) [![](w/page147a.gif)][P147] | [N3](#n3) [![](w/page169a.gif)][P169] | [N3](#n3) [![](w/page192a.gif)][P192] | [N3](#n3) [![](w/page211a.gif)][P211] | [N3](#n3)<br> [![](w/page234a.gif)][P234] |
```

A4 was mentioned in the user group in a pattern by Pierre Fouché. See also http://dentelles-de-javoison.over-blog.com/

Comment (MAETempels) [reply docs/index.md]: agreed

**Pull request merged**